content/en-us/reference/engine/enums/RollOffMode.yaml

```diff
@@ -16,20 +16,23 @@ items:
     summary: |
       Volume attenuates from `Class.Sound.RollOffMinDistance` in an inverse
       manner. This option mirrors how sounds attenuate in the real world.
+      This is done through `Class.Sound.RollOffMinDistance`/`Distance`
```

Comment:
```suggestion
      This is done through `Class.Sound.RollOffMinDistance`/`distance` where `distance` is the `Datatype.Vector3.Magnitude` between the audio source and the audio listener.
```

Reply:
How does this sound in terms of specifying `distance` without being overly technical, but also giving enough info that they know it's the vector magnitude between the source and listener?

Reply:
There should be enough context clues, I'll make a commit to add this in.